recipes/nspr/LICENSE

```diff
@@ -0,0 +1,373 @@
+Mozilla Public License Version 2.0
```

Comment:
There seems to be a license file included in the source (based on inspection). So we can drop this file.